node/src/components/linear_chain_sync/operations.rs

```diff
@@ -80,7 +76,7 @@ where
                         "Peer timed out",
                     );
                     // Peer timed out fetching the item, continue with the next peer
-                    continue;
+                    continue 'peer_loop;
```

Comment:
Why did you put `continue` in the `TimedOut` case explicitly but not in `Absent`?

Reply:
Good catch!  This `continue` doesn't appear to be necessary. In both the `TimeOut` case and the `Absent` case we'll continue with loop after the code moves on from this `match` block.